lnrpc/signrpc/signer.proto

```diff
@@ -0,0 +1,148 @@
+syntax = "proto3";
+
+package signrpc;
+
+message KeyLocator {
+    /// The family of key being identified.
+    int32 key_family = 1;
+
+    /// The precise index of the key being identified.
+    int32 key_index = 2;
+}
+
+message KeyDescriptor {
+     oneof key {
```

Comment:
This oneof construct is not working with walletkit.DeriveNextKey. It returns both the locator and the bytes.